_docs/getting-help.md

```diff
@@ -8,18 +8,19 @@ toc:
 nav: 7
 ---
 
-## Getting Help
+Getting Help
+------------
 
-### Getting Help as a user
-If you want to get started using Pico, please refer to the [user docs][HelpUserDocs] (you're reading them right now!). Please read the [upgrade notes][HelpUpgrade] if you want to upgrade from Pico 0.8 or 0.9 to Pico 1.0. You can find officially supported plugins and themes on [our website][OfficialPlugins]. A greater choice of third-party plugins and themes can be found in our [Wiki][] on the [plugins][WikiPlugins] or [themes][WikiThemes] pages respectively. If you want to create your own plugin or theme, please refer to the ["Getting Help as a developer" section](#getting-help-as-a-developer) below.
+### ... as a user
```

Comment:
Is there any reason to shorten this title? Sure, makes sense for the ToC, but in continues text it rather impedes readability.